contracts/SimpleAssetsExchange.sol

```diff
@@ -65,6 +85,10 @@ contract SimpleAssetsExchange is Ownable, IAssetsExchange {
     uint256 amountOut = _amount * priceProvider.getPrice(_asset) / 1 ether;
     require(amountOut > 0, "Incorrect output amount");
 
+    IERC20 token = IERC20(assetAddress[_asset]);
+    token.approve(address(pangolinRouter), tokenInAmount);
+    pangolinRouter.swapExactTokensForAVAX(tokenInAmount, minAmountOut, getPathForTokenToAVAX(asset), address(this), block.timestamp + 100);
+
     balance[msg.sender][_asset] = balance[msg.sender][_asset] - _amount;
 
     payable(msg.sender).transfer(amountOut);
```

Comment:
This won't work as the calculated amountOut may be different than the amount received from Pangolin